proto/process/agent.proto

```diff
@@ -718,4 +718,5 @@ message CollectorConnectionsTelemetry {
 	int64 connsBpfMapSize = 7;
 	int64 udpSendsProcessed = 8;
 	int64 udpSendsMissed = 9;
+	int64 conntrackSamplingPercent = 10;
```

Comment:
I haven't followed the convo around this data, is this expected to be a whole number?

Reply:
I believe so; [it's represented as an int64 in the netlink consumer](https://github.com/DataDog/datadog-agent/blob/38ba542b0837d9744a1699bf91208603db20552a/pkg/network/netlink/consumer.go#L77)